7024Movies/Pages/Testing.cshtml.cs

```diff
@@ -18,8 +18,8 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                string JsonString = webClient.DownloadString("https://api.themoviedb.org/3/search/movie?api_key=a852a3b3771672da86800503084b853b&language=en-US&query=star&page=1&include_adult=false");
-                var results = SearchResults.FromJson(JsonString);
+                string MovieReviews = webClient.DownloadString("https://api.themoviedb.org/3/search/movie?api_key=a852a3b3771672da86800503084b853b&language=en-US&query=star&page=1&include_adult=false");
+                var results = SearchResults.FromJson(MovieReviews);
```

Comment:
By this best practice it talks about why we should use a more concise word in naming variables. It helps in putting an allocating everything together. It also helps in preventing technical errors.

https://dzone.com/articles/best-practices-variable-and

Reply:
Good catch!  "JsonString" is easy, but could be anything.  "MovieReviews" is more descriptive and readable.